packages/wells/src/client/api/surveysApi.ts

```diff
@@ -0,0 +1,40 @@
+import { CDFHttpClient, HttpError} from '@cognite/sdk-core';
+import { SurveyData, SurveyDataRequest } from '../model/Survey';
+
+export class SurveysAPI {
+  private client?: CDFHttpClient;
+  private project?: String;
+
+  public set setHttpClient(httpClient: CDFHttpClient) {
+    this.client = httpClient;
+  }
+
+  public set setProject(project: String) {
+    this.project = project;
+  }
+
+  /* eslint-disable */
+  public getData = async (surveyId: number, start?: number, end?: number, limit?: number, cursor?: string, columns?: string[]): Promise<SurveyData | undefined> => {
```

Comment:
Yea, I wasn't quite sure what to decide on. Should we pass an object, or as parameters. If we pass as parameters, then it will probably be less code implementation for the once using the SDKs in the sense that they don't have to import data classes and instantiate data classes in their code to be able to use our methods. On the other hand, it might be a lot of pro's in passing data classes instead. What's your thought? Which one should we go for? @drsphelps